src/components/settings/timelapse/TimelapseSettings.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div>
+    <v-subheader id="timelapse">
+      {{ $t('app.timelapse.title.timelapse') }}
```

Comment:
`app.setting.title.timelapse` ?

Reply:
Don't see a need to duplicate this either? I think it's fine if it's changed to `app.general.title.timelapse` like suggested in your first comment.

Reply:
I do get your point, but I personally try to make little assumptions on this kind of stuff, as the same words can have different meanings depending on region and how they are presented on the screen.

Reply:
I think it's fine as-is for now, when more translations are added and there's conflicts this can always be changed